src/core/services/seo/BrandSeoService.ts

```diff
@@ -0,0 +1,26 @@
+import { SeoService } from './SeoService'
+import { Brand } from '../../types/product/BrandTypes'
+import { BrandMicroData } from './types'
+
+export class BrandSeoService extends SeoService {
+  data: Brand
+
+  constructor(data: Brand) {
+    super()
+    this.data = data
+  }
+
+  public getBrand(): string {
+    let brand = this.data
```

Comment:
creio que essa atribuição é desnecessária, pode usar somente `this.data`

Reply:
Desnecessária é, mas será que o código não fica mais limpo e mais fácil de entender?

Reply:
verdade acho que não é necessário